src/components/AddPost/index.tsx

```diff
@@ -87,6 +87,12 @@ const AddPost: VFC<AddPostType> = ({ onClose, isEditId, targetType, targetId })
         targetId,
       };
 
+      if ((!text || text === '') && !images.length && !files.length) {
```

Comment:
Just `!text` will suffice right?
cause `Boolean('') is false?
What if only `space` is entered as text?

Reply:
fixed